.pre-commit-config.yaml

```diff
@@ -1,23 +1,7 @@
 repos:
-  - repo: https://github.com/pycqa/autoflake
-    rev: v2.2.1
-    hooks:
-      - id: autoflake
-        args:
-          - "-i"
-          - "--remove-all-unused-imports"
-          - "--ignore-init-module-imports"
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-        - id: isort
-          name: isort (python)
-  - repo: https://github.com/psf/black
-    rev: 22.3.0
-    hooks:
-      - id: black
-        language_version: python3
-  - repo: https://github.com/pycqa/flake8
-    rev: 6.1.0
-    hooks:
-      - id: flake8
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.1.8
```

Comment:
I've seen that there is a more recent version. It may be a good idea to start with something up to date :)

Reply:
Oh okay okay, good.